nekoyume/Assets/_Scripts/UI/RankingBoard.cs

```diff
@@ -279,12 +277,8 @@ private void UpdateArena()
                 return;
             }
 
-            var (rank, arenaInfo) = _weeklyCachedInfo[0];
-            if (arenaInfo.Active)
-            {
-                currentAvatarCellView.Show((rank, arenaInfo, arenaInfo));
-            }
-            else
+            var (_, arenaInfo) = _weeklyCachedInfo[0];
```

Comment:
The first variable is not used. How about changing this?
`var cachedArenaInfo = _weeklyCachedInfo[0].arenaInfo;`

Reply:
Seems like a good idea. Fixed it!